src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -12,4 +16,5 @@ public Truck() {
     public void doWork() {
         System.out.println("Truck started to work");
     }
+
```

Comment:
The task description requires adding custom fields specific to a truck. Consider adding fields such as load capacity, number of axles, or any other relevant attributes that are unique to a truck.